src/coreclr/nativeaot/Runtime/eventpipe/CMakeLists.txt

```diff
@@ -9,15 +9,26 @@ set(AOT_EVENTPIPE_SHIM_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
 
 set (CONTAINER_SOURCES "")
 set (CONTAINER_HEADERS "")
+set (MINIPAL_SOURCES "")
+set (MINIPAL_HEADERS "")
 set (EVENTPIPE_SOURCES "")
 set (EVENTPIPE_HEADERS "")
 set (GEN_EVENTPIPE_SOURCES "")
 
 set (SHARED_CONTAINERS_SOURCE_PATH "${CLR_SRC_NATIVE_DIR}/containers")
 set (SHARED_EVENTPIPE_SOURCE_PATH "${CLR_SRC_NATIVE_DIR}/eventpipe")
+set (SHARED_MINIPAL_SOURCE_PATH "${CLR_SRC_NATIVE_DIR}/minipal")
 include (${SHARED_EVENTPIPE_SOURCE_PATH}/eventpipe.cmake)
 include (${SHARED_CONTAINERS_SOURCE_PATH}/containers.cmake)
 
+list(APPEND MINIPAL_SOURCES
+  utf8.c
+)
+
+list(APPEND MINIPAL_HEADERS
+  utf8.h
```

Comment:
This is not needed, all native code (repo-wide) has access to minipal, just use `#include<minipal/utf8.h>`.